docs/src/doc/decisions-and-ADRs/01_overview.stories.mdx

```diff
@@ -6,7 +6,7 @@ import { Meta } from '@storybook/addon-docs/blocks';
 
 Documentation on important decisions, non-tech and tech-related in the form of ADRs.
 
-## Wikit dev documentation
+## WiKit dev documentation
```

Comment:
Do you think this content should have its own section, at the same level as the _ADRs_ one? 🤔